--- origsrc/gtk-server-2.2.5/Makefile.in	2008-06-08 05:21:55.000000000 -0500
+++ src/gtk-server-2.2.5/Makefile.in	2008-06-17 18:14:06.625000000 -0500
@@ -1,3 +1,5 @@
+VPATH =		@srcdir@
+
 prefix =	@prefix@
 exec_prefix =	@exec_prefix@
 BINDIR =	@bindir@
@@ -6,7 +8,7 @@
 INCL =		@includedir@
 SYSCONFIG =	@sysconfdir@
 CC =		@CC@
-CFLAGS =	-Wall -pedantic @CFLAGS@ @CPPFLAGS@ @DEFS@ -DLOCALCFG=\"$(SYSCONFIG)/gtk-server.cfg\"
+CFLAGS =	-I. -Wall -pedantic @CFLAGS@ @CPPFLAGS@ @DEFS@ -DLOCALCFG=\"$(SYSCONFIG)/gtk-server.cfg\"
 LDFLAGS =	@LDFLAGS@ @LIBS@
 OBJS =		gtk-server.o @LIBOBJS@
 INSTALL = 	@INSTALL@
--- origsrc/gtk-server-2.2.5/configure.in	2008-06-08 05:21:55.000000000 -0500
+++ src/gtk-server-2.2.5/configure.in	2008-06-17 18:14:06.640625000 -0500
@@ -157,12 +157,7 @@
     AC_SUBST(BACKEND)
 fi
 
-# By default compile GTK-server binary
-if test "$win32" = yes; then
-    TARGET="gtk-server.exe"
-else
-    TARGET="gtk-server"
-fi
+TARGET="gtk-server$EXEEXT"
 DEFAULT="1"
 
 # Add compileflags for MacOSX
--- origsrc/gtk-server-2.2.5/gtk-server.cfg	2008-06-08 05:21:55.000000000 -0500
+++ src/gtk-server-2.2.5/gtk-server.cfg	2008-06-17 18:14:06.640625000 -0500
@@ -55,6 +55,30 @@
 #
 #---------------------------------------------------------------------------
 #
+# Cygwin
+#
+# GTK1 backend:
+# GTK_LIB_NAME = cyggtk-1-2-0.dll
+# GDK_LIB_NAME = cyggdk-1-2-0.dll
+# GLIB_LIB_NAME = cygglib-1-2-0.dll
+#
+# GTK2 backend:
+GTK_LIB_NAME = cyggtk-x11-2.0-0.dll
+GDK_LIB_NAME = cyggdk-x11-2.0-0.dll
+GLIB_LIB_NAME = cygglib-2.0-0.dll
+GOBJECT_LIB_NAME = cyggobject-2.0-0.dll
+ATK_LIB_NAME = cygatk-1.0-0.dll
+PANGO_LIB_NAME = cygpango-1.0-0.dll
+PIXBUF_LIB_NAME = cyggdk_pixbuf-2.0-0.dll
+GLADE_LIB_NAME = cygglade-2.0-0.dll
+#
+# XForms backend:
+# FORMS_LIB_NAME = cygforms-1.dll
+# FLIMAGE_LIB_NAME = cygflimage-1.dll
+# FORMSGL_LIB_NAME = cygformsGL-1.dll
+#
+#---------------------------------------------------------------------------
+#
 # Windows
 #
 # GTK1 backend:
